📝 The one that updates the design tokens docs.

sturobson opened the pull request:
🔥 removes quite a bit of design tokens documentation as it's now redundant as you should use the documentation in `vf-component-library`.
✨ introduces new gulp tasks (that might need a tidy up.
📝 updates design tokens documentation in `vf-component-library`.

sturobson committed:
updates vf-design tokens

sturobson committed:
updates design tokens documentation

sturobson committed:
re-compiled tokens

khawkins98 committed:
Add link to new documentation location

khawkins98 commented on tools/vf-component-library/gulpfile.js:
```diff
@@ -42,3 +47,33 @@ gulp.task('dev', gulp.series(
   'vf-build-search-index',
   gulp.parallel('watch','vf-watch')
 ));
+
```
Would be best if this was a separate gulp file (to keep separation of concerns). Happy to do that if you'd like.

sturobson replied:
that line? 

Happy for you to fix the gulp tasks I wrote into one `series` one and put it in the file system you've got for all the tasks.

khawkins98 replied:
Just added a commit that does a tidy and fixes a couple missing tags

https://github.com/visual-framework/vf-core/pull/1179/commits/88e4b003e8e8a1f42d83209b0136460aea72ae3e

khawkins98 committed:
Gulp token command tidy

- moves token tasks to a seperate js file
- removes unused dependencies
- adds token generation to dev and build commands
- fixes a couple minor things
- linting
- fix a missing `</main>` tag on theming page
- add vf-content to the index
